app/dao/organisation_dao.py

```diff
@@ -109,6 +109,7 @@ def dao_update_organisation(organisation_id, **kwargs):
 
     if 'email_branding_id' in kwargs:
         _update_organisation_services(organisation, 'email_branding')
+        dao_add_email_branding_to_organisation_pool(organisation_id, kwargs['email_branding_id'])
```

Comment:
If the branding chosen to be the default is the GOV.UK branding, then `email_branding_id` is `None` and the dao function gives an error.